openpype/pipeline/README.md

```diff
@@ -0,0 +1,29 @@
+# Colorspace
```

Comment:
I feel this is the wrong place for this - this should just be in the DEV docs. No?

Reply:
Yeah, I believe you are right - for some reason I have missed that section of the documentation..

Reply:
so I have move it to Dev documentation.